tools/etcd-dump-db/backend.go

```diff
@@ -125,6 +127,14 @@ func authUsersDecoder(k, v []byte) {
 	fmt.Printf("user=%q, roles=%q, option=%v\n", user.Name, user.Roles, user.Options)
 }
 
+func metaDecoder(k, v []byte) {
+	if string(k) == string(schema.MetaConsistentIndexKeyName) || string(k) == string(schema.MetaTermKeyName) {
+		fmt.Printf("key=%q, value=%v\n", k, binary.BigEndian.Uint64(v))
+	} else {
+		fmt.Printf("key=%q, value=%q\n", k, v)
```

Comment:
This is a fallback to default decoder from line 160. We should ensure that this code lines match. Can you create a function `defaultDecoder` and use in both places?

Reply:
Done & Thanks.